defender/governance_summary/README.md

```diff
@@ -4,16 +4,15 @@ This Defender Autotask will periodically check and display the status of active
 
 ## Optimizations
 
-During the first run of the Autotask, it will scan from block zero to the latest block.  It will then search for new `ProposalCreated` events starting from the last block searched.  The current status of all active proposals will then be displayed in the Autotask logs.
-
+During the first run of the Autotask, it will scan from block zero to the latest block.  It will then search for new `ProposalCreated` events starting from the last block searched. The current status of all active proposals will then be displayed in the Autotask logs.
```

Comment:
Inconsistent spacing separating sentences in same paragraph.  I've seen both single and double used elsewhere in this repo. I prefer double spaces.